opm/grid/common/ZoltanPartition.cpp

```diff
@@ -103,19 +104,51 @@ zoltanGraphPartitionGridOnRoot(const CpGrid& cpgrid,
     int                         rank  = cc.rank();
     std::vector<int>            parts(size, rank);
     std::vector<std::vector<int> > wells_on_proc;
+    // List entry: process to exort to, (global) index, attribute there (not needed?)
+    std::vector<std::tuple<int,int,char>> myExportList(numExport);
+    // List entry: process to import from, global index, attribute here, local index
+    // (determined later)
+    std::vector<std::tuple<int,int,char,int>>myImportList(numImport);
+    myExportList.reserve(.2*myExportList.size());
```

Comment:
.2 ?

Reply:
Well that is just a guess. In most cases `myExportList.size()` will not be enough we reserve a bit more and hope to prevent copying.

Reply:
reserve is not additive, you are reserving less items than what already exists in the vector. maybe you meant importList for one of them?

Reply:
thanks for the persistence. So embarassing. Fixed.